test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1

```diff
@@ -17,6 +17,13 @@ Describe "Start-Sleep DRT Unit Tests" -Tags "CI" {
         $watch.Stop()
         $watch.ElapsedMilliseconds | Should BeGreaterThan 950
     }
+
+    It "Should work properly when sleeping with ms alias" {
+        $watch = [System.Diagnostics.Stopwatch]::StartNew()
+        Start-Sleep -ms 1000
+        $watch.Stop()
+        $watch.ElapsedMilliseconds | Should BeGreaterThan 950
```

Comment:
It's not a big problem, people mostly never remove code anyway and if they did, it should be caught during the PR, but it's a nice idea.